Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -2203,12 +2187,6 @@ script:
     - contextPath: Gmail.ForwardingAddress.forwardingEmail
       description: An email address to which messages can be forwarded.
       type: String
-    - contextPath: Gmail.ForwardingAddress.Disposition
-      description: The state that a message should be left in after it has been forwarded.
-      type: String
-    - contextPath: Gmail.ForwardingAddress.Enabled
```

Comment:
same question

Reply:
there is only one contextPath of Gmail.ForwardingAddress.Enabled in the update command